services/QuillLMS/client/app/bundles/Staff/utils/evidence/genAIAPIs.ts

```diff
@@ -31,6 +31,19 @@ export const fetchDatasetAndAssociatedRecords = async ({ queryKey }) => {
   };
 }
 
+export const fetchTrialAndAssociatedRecords = async ({ queryKey }) => {
+  const [key, datasetId, trialId]: [string, string, string] = queryKey
+
+  const response = await researchGenAiApiFetch(`datasets/${datasetId}/trials/${trialId}`);
+
+  let data = await response.json();
+
+  return {
+    error: handleApiError('Failed to fetch trial, please refresh the page.', response),
+    data
+  };
```

Comment:
I don't actually have a strong thought here, but this return pattern seems pretty common for our error handling.  I wonder if there's some way to avoid the repetition.  Of course, these aren't actually identical, so maybe it's more trouble than it's worth.